src/configuration/FirestoreConfigurationContext.tsx

```diff
@@ -150,3 +181,43 @@ function getUserProfilesFromSnapshot(
   }
   return undefined;
 }
+
+function getModesWeSellTicketsForFromSnapshot(
+  snapshot: FirebaseFirestoreTypes.QuerySnapshot,
+): string[] | undefined {
+  return snapshot.docs
+    .find((doc) => doc.id == 'other')
+    ?.get<string[]>('modesWeSellTicketsFor');
+}
+
+function getPaymentTypesFromSnapshot(
+  snapshot: FirebaseFirestoreTypes.QuerySnapshot,
+): PaymentType[] | undefined {
+  let paymentTypesField = snapshot.docs
+    .find((doc) => doc.id == 'paymentTypes')
+    ?.get<string[]>('app');
+  if (paymentTypesField != undefined) {
+    return mapPaymentTypeStringsToEnums(paymentTypesField);
+  }
+  return undefined;
+}
+
+function mapPaymentTypeStringsToEnums(
+  arrayOfPaymentTypes: string[],
+): PaymentType[] {
+  var paymentTypes: PaymentType[] = [];
+  for (const paymentType of arrayOfPaymentTypes) {
+    switch (paymentType) {
+      case 'vipps':
+        paymentTypes.push(PaymentType.Vipps);
+        break;
+      case 'visa':
+        paymentTypes.push(PaymentType.VISA);
+        break;
+      case 'mastercard':
+        paymentTypes.push(PaymentType.MasterCard);
+        break;
+    }
```

Comment:
There is another way to match those payment methods but for example `MasterCard` doesn't match `mastercard` key in the `Firestore`. If it is possible to match seamless what we have in the Firestore vs the enum keys we could avoid that `switch` block. @gorandalum @monopoint

Reply:
Good suggestion. Feel free to change the naming of variables in the code the handle this mapping better.